pages/_app.tsx

```diff
@@ -32,7 +33,7 @@ const App: FC<{
 }> = ({ Component, pageProps }) => {
   const relayCache = pageProps.__relayCache as RecordMap | undefined;
 
-  const { asPath } = useRouter();
+  const locationKey = useStabilizedRouteTransitionKey();
```

Comment:
rather than simply using the route's path to determine when to trigger the fade transition, this new hook makes it more strict.

for example, when we transition between NFT detail views in the modal, each view is technically a new route path. but we don't want to trigger a full-screen fade in that scenario.